### DISCUSSION TITLE
The generated image does not appear

### DISCUSSION THREAD

**Discussion by Ilja-HP** [vladmandic/sdnext#811] [Q&A]
The generated image does not appear, I have reinstalled vladmandic SD for the third time and do not understand what the problem is. My PC has Python 3.10.7 and GPU gtx 1660TI, 16gb RAM installed. I switched from AUTOMATIC1111 due to the fact that there are constant problems with a lack of VRAM memory, the loss:nan problem, and all problems appear unexpectedly and suddenly, as if AUTOMATIC1111 lives its own life, I don’t change the settings and use those recommended in popular tutorials. Help to understand please.

**Comment by TomerGadol** [comment]
Is it possible you are getting black images and therefore it appears as if there is no image? Check your outputs folder.
I also have 1660ti and for some reason, I only get black images. Trying to tinker with the settings right now.

**Comment by vladmandic** [comment]
Read through #374
